src/Service.Tests/dab-config.CosmosDb_NoSql.json

```diff
@@ -148,6 +148,28 @@
         }
       }
     },
+    "Sun": {
+      "source": "graphqldb.sun",
+      "permissions": [
+        {
+          "role": "anonymous",
+          "actions": [
+            "create",
+            {
+              "action": "read",
+              "fields": {
+                "include": [
+                  "*"
+                ]
+              }
+            },
+            "update",
+            "delete"
+          ]
+      }
+      ],
+      "graphql": true
```

Comment:
Can you double check if there are any other combinations or possibilities that needs to be tested here?